include/SDL3/SDL_process.h

```diff
@@ -0,0 +1,157 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+/**
+ * # CategoryProcess
+ *
+ * Process control support.
+ */
+
+#ifndef SDL_process_h_
+#define SDL_process_h_
+
+#include <SDL3/SDL_error.h>
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    /** Create a pipe to the process' stdin. */
+    SDL_PROCESS_STDIN = 1 << 0,
+    /** Create a pipe from the process' stdout. Without this option, the process will output to the parent's stdout. */
+    SDL_PROCESS_STDOUT = 1 << 1,
+    /** Create a pipe from the process' stderr. Without this option, the process will output to the parent's stderr. */
+    SDL_PROCESS_STDERR = 1 << 2,
+    /** Allow SDL to report errors on the child process' stderr if launching the process failed after a fork(). */
+    SDL_PROCESS_ERRORS_TO_STDERR = 1 << 3,
+} SDL_ProcessFlags;
+
+typedef struct SDL_Process SDL_Process;
```

Comment:
Calling this "Subprocess", like Python `subprocess` and GLib `GSubprocess`, might be better - that will give it an better indication that this API is specifically for forking a direct child process, and is not for managing arbitrary other processes.

Reply:
I also prefer `subprocess`. "process" means the current process, and `subprocess` means the child processes.